### DISCUSSION TITLE
KusionStack + KCL 社区中文周会 (2024.02.22)

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#17] [General]
腾讯会议列表：778-2381-6338

**Comment by Peefy** [comment]
Record

+ KCL 最新动态内容同步 https://kcl-lang.io/blog/2024-02-22-newsletter，发布 0.8 alpha 版本，预计下周产出正式版本
+ KCL 0.9 Milestone 已确定事项同步 https://github.com/kcl-lang/kcl/milestone/9，其他事项更新中
+ Kusion 0.11 版本迭代计划同步，主要包含 Kusion Module 的进一步开放性和命令行工具改进
+ LFX KCL 题目和进展同步：https://mp.weixin.qq.com/s/3i1SiA4RdIhHHDU_SDHblg

**Reply by hoposicufo** [reply]
> Record
> 
> * KCL 最新动态内容同步 [https://kcl-lang.io/blog/2024-02-22-newsletter，发布](https://kcl-lang.io/blog/2024-02-22-newsletter%EF%BC%8C%E5%8F%91%E5%B8%83) 0.8 alpha 版本，预计下周产出正式版本
> * KCL 0.9 Milestone 已确定事项同步 [https://github.com/kcl-lang/kcl/milestone/9，其他事项更新中](https://github.com/kcl-lang/kcl/milestone/9%EF%BC%8C%E5%85%B6%E4%BB%96%E4%BA%8B%E9%A1%B9%E6%9B%B4%E6%96%B0%E4%B8%AD)
> * Kusion 0.11 版本迭代计划同步，主要包含 Kusion Module 的进一步开放性和命令行工具改进
> * LFX KCL 题目和进展同步：https://mp.weixin.qq.com/s/3i1SiA4RdIhHHDU_SDHblg

KCL en de bijbehorende tools (Kusion) worden actief ontwikkeld met regelmatig nieuwe functies en verbeteringen.

Dit is geweldig nieuws voor iedereen die KCL gebruikt of overweegt te gebruiken voor configuratie als code of beleid als code. Besuchen Sie https://viggoslots-de.de/ für die neuesten Angebote, attraktive Bonusprogramme und unser breites Spielangebot.  Het toont een gezonde en actieve ontwikkelingsgemeenschap.
